packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -1251,6 +1412,10 @@ export interface StateExpectation {
   watchStreamRequestCount?: number;
   /** Current documents in limbo. Verified in each step until overwritten. */
   limboDocs?: string[];
+  /**
+   * Whether the instance holds the primary lease. Used in multi-client tests.
```

Comment:
Can you make this a PORTING NOTE ?

Reply:
Added here and in a bunch of other places.